shipment_advice/models/stock_picking.py

```diff
@@ -157,6 +160,12 @@ def _compute_shipment_loaded_progress(self):
                 picking.loaded_weight_progress = (
                     f"{picking.loaded_weight} / {total_weight}"
                 )
+            waiting_moves = picking.move_lines.filtered(
```

Comment:
Any chance to have a test?

Reply:
A bit late, but indeed that would be nice, especially because qty fields tend to be renamed on each Odoo version :stuck_out_tongue_closed_eyes:

Reply:
I probably did not add the test, because that compute method I inserted my change, was not included in the existing unit tests...

Reply:
Test added :check